hackmcgill/hackerAPI-frontend
# Feature/252 search

# Part 1 of search feature (#252)
## Includes the following:

- #251: filtering hackers based off of school, graduation year, degree, status, skill, and job search
- #21: export search result to CSV

## Does *NOT* include the following:

- Update status

## Figma

https://www.figma.com/file/TU0ycyxLagQ8xio2GhfDKqMj/Registration-system-desktop?node-id=638%3A354

## Screenshots

<img width="1680" alt="screen shot 2018-12-26 at 3 26 19 am" src="https://user-images.githubusercontent.com/17463268/50429365-43741400-08be-11e9-8ccc-96f3c04c9394.png">

![screen shot 2018-12-26 at 3 27 41 am_censored](https://user-images.githubusercontent.com/17463268/50456202-25a4be80-0953-11e9-943e-2fe23c4bb079.jpg)

## How to test

- Verify that /api/search is in STAFF role
- Log in using staff account
- Access http://localhost:1337/admin/search
